docs/content/concepts/layer/validation/_index.md

```diff
@@ -0,0 +1,61 @@
+
+If you are not an OSCAL tool developer, this technical information might be useful either if you seek general understanding of OSCAL, or if as a tool *user* you encounter them in your work.
+
+## Why are these concepts important?
+
+It is important that OSCAL tool developers implement software that confirm the JSON, XML, and YAML is well-formed and the [OSCAL document instances](https://pages.nist.gov/OSCAL/concepts/layer/) are valid.
+
+Generally, both "validity" and "well-formedness" define OSCAL, such that a conformant OSCAL application can respect them, and an application that does not is *ipso facto* not an OSCAL application.
+
+## What do "well-formed" and "valid" mean to the OSCAL community?
+
+When developing tools to read or write OSCAL, NIST or fellow community developers will ask if your data is *well-formed* and if the OSCAL document is *valid*. We define these terms below and explain how current OSCAL tools support checking well-formed data and valid OSCAL.
+
+### What is well-formed data?
+
+*Well-formed* means that your JSON, XML, YAML (whether it is OSCAL or not) follows the syntax rules of that format.
+
+#### What OSCAL tools help you check well-formedness?
+
+NIST's OSCAL tools do not explicitly check that JSON, XML, or YAML is well-formed. If you are a tool developer, we expect you to use your language runtime and libraries to parse JSON, XML, and YAML and check for the well-formed data. This must occur before validating an OSCAL document.
+
+### What is a valid OSCAL document?
+
+*Valid* means that your OSCAL document (in a JSON, XML, or YAML already checked for *well-formedness*) follows the rules of a model. A model can be expressed using a schema format, such as [JSON Schema](https://json-schema.org/) or [XML Schema](https://www.w3.org/XML/Schema).
+
+### Does OSCAL provide schemas for OSCAL models?
+
+The NIST OSCAL Team develops and maintains [JSON Schema](https://github.com/usnistgov/OSCAL/tree/main/json/schema) to validate OSCAL documents in JSON and YAML data formats and [XML Schema](https://github.com/usnistgov/OSCAL/tree/main/xml/schema) to validate OSCAL documents in the XML data format.
+
+
+### What tools check that an OSCAL document is valid?
+
+There are many tools to validate documents using schema formats.
+
+Here are some Internet resources to help you find a tool.
+
+- [JSON Schema validator](https://json-schema.org/implementations.html#validators)
+- [XML Schema validators](https://www.w3.org/XML/Schema#Tools)
+
+**Note: These lists are not exaustive and NIST does not endorse any of these tools.**
+
+### Why can OSCAL document validation fail?
+
+There are multiple failure scenarios for OSCAL documents, and JSON, XML, and YAML more generally:
+
+1. The document is well-formed, but is not conformant to the schema and errors are correctly reported.
+2. The document is not well-formed, so validation cannot occur.
```

Comment:
Minor observation: lack of well-formed precludes schema validation, so well-formedness should precede schema validation.

Reply:
Don't we cover that later int he document?

> #### What OSCAL tools help you check well-formedness?

> NIST's OSCAL tools do not explicitly check that JSON, XML, or YAML is well-formed. If you are a tool developer, we expect you to use your language runtime and libraries to parse JSON, XML, and YAML and check for the well-formed data. **This must occur before validating an OSCAL document.** (Emphasis add here by me for context.)

Reply:
I can see your point, but I do not say whether it is logically or temporally before (if we are addressing the snippet where I copied from).

What do you recommend we add or take away?

I only added this bit to address the question in the section header: Q: What OSCAL tools [from NIST] check well-formedness? A: NIST's OSCAL tools do not provide tools to check well-formedness, because if you are a tool developer it is on you to check that before (logically or temporally) using schemas to validate. [This implies: do not report issues back to NIST with problems in schema validation when it is easy to spot the document isn't well-formed. These are not schema issues, it should be obvious but you ought to check the doc is well-formed somewhere in your code.]

So, how can we write that better?